_topic_map.yml

```diff
@@ -245,6 +245,7 @@ Name: Container Security Guide
 Dir: security
 Distros: openshift-enterprise,openshift-origin
 Topics:
+<<<<<<< HEAD
```

Comment:
@tmorriso-rh - Hey Traci - the issue is that not all merge conflicts have been resolved before pushing this commit.